imagekitio/url.py

```diff
@@ -192,8 +195,8 @@ def transformation_to_str(transformation):
                         value = value.strip("/")
                         value = value.replace("/", "@@")
                     if transform_key == "raw":
-                        for i in value.split(","):
-                            parsed_transform_step.append(i)
+                        for j in value.split(","):
+                            parsed_transform_step.append(j)
```

Comment:
Test case missing for this change. Is it related to `ik-attachment`?

Reply:
It is related to `transformation['raw']` issue #44.

Reply:
Added test case. PR updated.